roles/elastic-stack/ansible-elasticsearch/tasks/main.yml

```diff
@@ -67,9 +67,9 @@
   when:
     - elasticsearch_xpack_security
 
-- name: Configure Elasticsearch.
-  template:
-    src: elasticsearch.yml.j2
+- name: Fetch latest Wazuh alerts template
+  get_url:
+    url: https://github.com/wazuh/wazuh/blob/master/extensions/elasticsearch/7.x/wazuh-template.json
```

Comment:
`master` should be a tag according to versions specified

Reply:
Done on d82d1b51d149ee4f1d9f9f66b2566e37ae9800de